CefSharp/DependencyChecker.cs

```diff
@@ -0,0 +1,200 @@
+            }
+
+            // Loop through CefSharp dependencies and add to list if not found
+            foreach (var cefSharpDependency in CefSharpDependencies)
+            {
+                var dependencyPath = Path.Combine(path, cefSharpDependency);
+
+                if (!File.Exists(dependencyPath))
+                {
+                    missingDependencies.Add(cefSharpDependency);
+                }
+            }
+
+            //If path path is not rooted (doesn't start with a drive letter + folder)
+            //then make it relative to the executing assembly.
+            var localePath = Path.IsPathRooted(localePackFile) ? localePackFile : Path.Combine(path, localePackFile);
+
+            if (!File.Exists(localePath))
+            {
+                missingDependencies.Add(localePackFile);
+            }
+
+            return missingDependencies;
+        }
+
+        /// <summary>
+        /// Checks if all Cef and CefSharp dependencies were found relative to the Executing Assembly.
+        /// Shortcut method that calls <see cref="CheckDependencies"/>, throws an Exception if not files are missing.
+        /// </summary>
+        /// <param name="locale">The locale, if empty then en-US will be used.</param>
+        /// <param name="localesDirPath">The path to the locales directory, if empty locales\ will be used.</param>
+        /// <param name="resourcesDirPath">The path to the resources directory, if empty the Executing Assembly path is used.</param>
+        /// <param name="packLoadingDisabled">Is loading of pack files disabled?</param>
+        /// <exception cref="Exception">Throw when not all dependencies are present</exception>
+        public static void AssertAllDependenciesPresent(string locale, string localesDirPath, string resourcesDirPath, bool packLoadingDisabled)
+        {
+            var executingAssembly = Assembly.GetExecutingAssembly();
+
+            var path = Path.GetDirectoryName(executingAssembly.Location);
+
+            if(string.IsNullOrEmpty(locale))
+            {
+                locale = "en-US";
+            }
+
+            if (string.IsNullOrEmpty(localesDirPath))
+            {
+                localesDirPath = @"locales";
+            }
+
+            if (string.IsNullOrEmpty(resourcesDirPath))
+            {
+                resourcesDirPath = path;
+            }
+
+            var missingDependencies = CheckDependencies(true, packLoadingDisabled, path, resourcesDirPath, Path.Combine(localesDirPath, locale + ".pak"));
+
+            if (missingDependencies.Count > 0)
+            {
+                var builder = new StringBuilder();
+                builder.AppendLine("Unable to locate required Cef/CefSharp dependencies:");
+
+                foreach (var missingDependency in missingDependencies)
+                {
+                    builder.AppendLine("Missing:" + missingDependency);
+                }
+
+                builder.AppendLine("Executing Assembly Path:" + path);
+
```

Comment:
We could also add an informational reminder in this string about locales\xxxx.pak that explains what they do if we know even though I would never suggest checking for the dependency to exist.